src/app/helptext/system/certificates.ts

```diff
@@ -293,7 +302,14 @@ certificate still approved.'),
     key_usage: {
       config: {
         placeholder: T('Key Usage Config'),
-        tooltip: T('The key usage extension defines the purpose \
+        tooltip: T('Specify the valid key usages for this certificate.\
```

Comment:
I recommend:

`Specify this certificate's valid Key Usages. Web certificates typically need at least Digital Signature and possibly Key Encipherment or Key Agreement, while other applications may need other usages.`